services/brig/src/Brig/API.hs

```diff
@@ -1547,20 +1547,18 @@ canBeDeletedH (uid ::: tid) = do
     onlyOwner <- lift (Team.teamOwnershipStatus uid tid)
     case onlyOwner of
```

Comment:
I would consider this case distinction logic, so it could live in a separate `canBeDeleted` function.

Reply:
351e9d4c 

(not sure which is better; now there are some implicit assumptions in both cases about the *reasons* for the boolean.  but i think either one is good enough.)